component/input/input.css

```diff
@@ -0,0 +1,28 @@
+div.input_box {
```

Comment:
are u using here div for specificity, if not then you can remove this.

Reply:
yes for specificity only it's bad practice I think I will correct it